specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/ServerAzureADAdministrators.json

```diff
@@ -0,0 +1,395 @@
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found\n\n * 404 ServerNotInSubscriptionResourceGroup - Specified server does not exist in the specified resource group and subscription.\n\n * 404 ResourceNotFound - The requested resource was not found."
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Gets a list of Azure Active Directory administrator.": {
+            "$ref": "./examples/AdministratorList.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AdministratorProperties": {
+      "description": "Properties of a active directory administrator.",
+      "required": [
+        "administratorType",
+        "login",
+        "sid"
+      ],
+      "type": "object",
+      "properties": {
+        "administratorType": {
+          "description": "Type of the sever administrator.",
+          "enum": [
+            "ActiveDirectory"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "AdministratorType",
+            "modelAsString": true
+          }
+        },
+        "login": {
+          "description": "Login name of the server administrator.",
+          "type": "string"
+        },
+        "sid": {
+          "format": "uuid",
+          "description": "SID (object ID) of the server administrator.",
+          "type": "string"
+        },
+        "tenantId": {
+          "format": "uuid",
+          "description": "Tenant ID of the administrator.",
+          "type": "string"
+        }
+      }
+    },
+    "Resource": {
+      "description": "ARM resource.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Resource ID.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "Resource name.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "Resource type.",
+          "type": "string",
+          "readOnly": true
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "ProxyResource": {
+      "description": "ARM proxy resource.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {}
+    },
+    "ServerAzureADAdministrator": {
+      "description": "Azure Active Directory administrator.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/AdministratorProperties",
+          "description": "Resource properties.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "AdministratorListResult": {
```

Comment:
For compatibility this should be ServerAdministratorListResult

https://github.com/Azure/azure-rest-api-specs/blob/master/specification/sql/resource-manager/Microsoft.Sql/stable/2014-04-01/serverAzureADAdministrators.json#L258